docs.yaml

```diff
@@ -102,3 +118,30 @@ versions:
     ref: '3.1'
   - name: '3.0'
     ref: '3.0'
+rewrites:
+  - search: 'https://(helpdocs|docs).datastax.com/(en/astra/)?(aws|gcp)/(dscloud/apollo|doc/dscloud/astra)/dscloudGettingStarted.html'
+    replace: 'https://docs.datastax.com/en/astra-serverless/docs/getting-started/getting-started.html'
+  - search: 'https://(helpdocs|docs).datastax.com/(en/astra/)?(aws|gcp)/(dscloud/apollo|doc/dscloud/astra)/dscloudObtainingCredentials.html'
+    replace: 'https://docs.datastax.com/en/astra-serverless/docs/connect/secure-connect-bundle.html'
+  - search: 'https://(helpdocs|docs).datastax.com/(en/astra/)?(aws|gcp)/(dscloud/apollo|doc/dscloud/astra)/dscloudShareClusterDetails.html'
+    replace: 'https://docs.datastax.com/en/astra-serverless/docs/manage/org/manage-tokens.html'
+  - search: 'http://downloads.datastax.com/java-driver/'
+    replace: 'https://downloads.datastax.com/#datastax-drivers'
+  - search: 'https://docs.datastax.com/en/drivers/java/(4.[0-9]+)/com/datastax/oss/driver/api/mapper/EntityHelper.html'
+    replace: 'https://docs.datastax.com/en/drivers/java/\1/com/datastax/oss/driver/api/mapper/entity/EntityHelper.html'
+  - search: '(http|https)://www.datastax.com/drivers/java/'
+    replace: 'https://docs.datastax.com/en/drivers/java/'
+  - search: 'http://docs.datastax.com/en/drivers/java'
+    replace: 'https://docs.datastax.com/en/drivers/java'
+  - search: 'https://docs.astra.datastax.com/docs/creating-your-astra-database'
+    replace: 'https://docs.datastax.com/en/astra-serverless/docs/getting-started/create-db-choices.html'
+  - search: 'https://docs.astra.datastax.com/docs/obtaining-database-credentials'
+    replace: 'https://docs.datastax.com/en/astra-serverless/docs/connect/secure-connect-bundle.html'
+  - search: 'https://docs.datastax.com/en/drivers/java/(4.[0-9]+)/com/datastax/oss/driver/api/core/cql/SyncCqlSession.html%60'
+    replace: 'https://docs.datastax.com/en/drivers/java/\1/com/datastax/oss/driver/api/core/cql/SyncCqlSession.html'
+  - search: 'https://code.google.com/p/snappy/'
+    replace: 'https://google.github.io/snappy/'
+  - search: 'https://code.google.com/p/guava-libraries/wiki/ListenableFutureExplained'
+    replace: 'https://github.com/google/guava/wiki/ListenableFutureExplained'
+  - search: 'https://community.datastax.com/index.html'
+    replace: 'https://www.datastax.com/workshops'
```

Comment:
There's a lot of these that are directly referencing the java driver, either they need to be modified to refer to the csharp driver or deleted.

Reply:
@emeliawilkinson24 This will apply to all of the PRs in other driver builds.

Reply:
When does it make sense to replace the link in the doc files themselves vs adding a rewrite here in the docs.yaml?